### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ivansafiulin**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0px;
+}
+
+[class*="item-"] {
+  height: 100px;
+  width: 100%;
```
Comment (gestap) [inline src/style.css]: it's a default value for `div`, no need to write it.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.item-1 {
```
Comment (gestap) [inline src/style.css]: `item-1` bad name for class

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+.item-2 {
+  background: green;
+}
+
+.item-3 {
+  background: blue;
+}
+
+
```
Comment (gestap) [inline src/style.css]:
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.item-2 {
+  background: green;
+}
+
+.item-3 {
+  background: blue;
+}
+
```
Comment (gestap) [inline src/style.css]:
```suggestion
```

File: src/style.css (hunk 3 above)
Comment (gestap) [reply src/style.css]: you need only one empty line on the end of the document

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0px;
```
Comment (gestap) [inline src/style.css]: if you write `0`, no need to write `px, rem ...`, it`s a mistake for browser.
```suggestion
  margin: 0;
```

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+[class*="item-"] {
```
Comment (gestap) [inline src/style.css]: it's good but I recommend to use class. It will be easier to read and debug code.

File: src/style.css (hunk 1 above)
Comment (gestap) [reply src/style.css]: remove `width`, `div` already have it.

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+.block {
+  height: 100px;
+  width: 100%;
+}
+
+.red.block {
```
Comment (gestap) [inline src/style.css]: No need to write like this. Better will be `.red`

File: src/index.html [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="red block"></div>
```
Comment (gestap) [inline src/index.html]:
```suggestion
  <div class="block-red block"></div>
```

**Comment by ivansafiulin** [comment]
.

**Pull request closed without merging**